lib/spack/docs/basic_usage.rst

```diff
@@ -1222,11 +1026,11 @@ Module Commands for Shell Scripts
 
 Although Spack is flexible, the ``module`` command is much faster.
 This could become an issue when emitting a series of ``spack load``
-commands inside a shell script.  By adding the ``--shell`` flag,
-``spack module find`` may also be used to generate code that can be
-cut-and-pasted into a shell script.  For example:
+commands inside a shell script.  The ``spack module loads`` may also
+be used to generate code that can be cut-and-pasted into a shell
+script.  For example:
 
-.. code-block:: console
+.. code-block:: sh
```

Comment:
This is probably just a rebase mistake, but this code-block needs to be formatted as console output and not a bash script.